internal/v2/cert_verifier/cert_verifier_test.go

```diff
@@ -65,6 +67,16 @@ func TestVerifyClientCertChain(t *testing.T) {
 		log.Fatalf("error starting fake S2Av2 Server: %v", err)
 	}
 
+	os.Setenv("S2A_ACCESS_TOKEN", "token")
```

Comment:
In each unit test, please use a different token value (for example, in this test it could be "TestVerifyClientCertChain_s2a_access_token"). This helps prevent any dependence between the tests and make sure we reset everything correctly between each test.